app/assets/javascripts/drafts.coffee

```diff
@@ -76,6 +76,11 @@ $(document).ready ->
     e.stopImmediatePropagation()
 
   incrementElementIndex = (newDiv, multipleIndex, simple, type) ->
```

Comment:
The good news is, it is also doing it in master, so I didn't break it! At least not in the branch haha

I'm looking into it